deploy/kubernetes/overlays/dev/kustomization.yaml

```diff
@@ -10,8 +10,7 @@ patchesStrategicMerge:
 
 namespace: gcp-filestore-csi-driver
 
-# To change the dev image, add something like the following.
-#images:
-#- name: gcr.io/gke-release/gcp-filestore-csi-driver
-#  newName: <your-gcp-project-image-url>
-#  newTag: <imagetag>
+images:
+- name: registry.k8s.io/cloud-provider-gcp/gcp-filestore-csi-driver
+  newName: gcr.io/riteshghorse-consumer/gcp-filestore-csi-driver
```

Comment:
you probably don't want to include changes in this file.

Reply:
yeah looks like I accidentally committed these files yaml files